src/model_getters.jl

```diff
@@ -10,6 +10,17 @@ function moisture_model(parsed_args)
     end
 end
 
+function model_config(parsed_args)
+    config = parsed_args["config"]
+    return if config == "sphere"
+        SphericalModel()
+    elseif config == "column"
+        SingleColumnModel()
+    elseif config == "box"
+        BoxModel()
+    end
+end
```

Comment:
Can we split this out into a separate PR?

Reply:
#1041